src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -331,24 +331,493 @@ The following workflows represent some mathematical functions that can be optimi
+| URL
+| `API_EXTENSION`
+| the base path to access the deployment endpoint.
+| String (default="/api/deploy")
+| `MODEL_URL`
+| the URL of the model that need to be deployed.
+| URL
+| `SERVICE_TOKEN`
+| the api token to be able to deploy the model.
+| String
+|===
+
+
+=== Call the service for predictions
+
+There is two ways to call the web service to score input data using MLOS:
+
+*Via Swagger UI*
+
+Open the swagger UI.
+
+Open the get_token` operation and get an api_token by entering your username (default value is `user`).
+
+Open the `predict` operation and set the provided token and the data that you need to score.
+
+image::MAS_Predict_Swagger.png[align=center]
+
+* *via Studio portal*
+
+Open the link:https://try.activeeon.com/Studio[Studio Portal].
+
+Create a new workflow.
+
+Add the `model_as_a_service` bucket.
+
+Drag and drop the `Call_Prediction_Service` task from the bucket.
+
+Execute the Workflow and set the different workflow variables as following:
+
+.Call_Prediction_Service variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `PREDICT_MODEL_ENDPOINT`
+| the endpoint of the started service.
+| URL
+| `PREDICT_EXTENSION`
+| the base path to access the prediction endpoint.
+| String (default="/api/predict")
+| `SERVICE_TOKEN`
+| the api token to be able to deploy the model.
+| String
+| `INPUT_DATA`
+| entry data that needs to be scored by the deployed model.
+| JSON
+| `LABEL_COLUMN`
+| need to be set if data is labeled.
+| String (default='class')
+|===
+
+=== Delete the service
+
+There is two ways to delete a web service (i.e. where you will deploy, update and call your model) using MLOS:
+
+*Via Cloud Automation Portal*
+
+Open the link:https://try.activeeon.com/automation-dashboard/#/portal/cloud-automation[Cloud Automation Portal].
+
+Set the action `Finish_Service` and click on execute.
+
+image::MAS_Delete_Service.png[align=center]
+
+*Via Studio portal*
+
+Open the link:https://try.activeeon.com/Studio[Studio Portal].
+
+Create a new workflow.
+
+Add the `model_as_a_service` bucket.
+
+Drag and drop the `Model_Service_Actions` task from the bucket.
+
+Execute the Workflow and set the different workflow variables as following:
+
+.Model_Service_Actions variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `ACTION`
+| The action that will be processed regarding the service status.
+| List (Pause_Model_Service, Resume_Model_Service, Finish_Model_Service) (default="Finish_Model_Service")
+| `INSTANCE_NAME`
+| the service name that need to be deleted, resumed or paused.
+| String (default="model-server-${PA_JOB_ID}")
+| `INSTANCE_ID`
+| the service instance id
+| String
+|===
+
+=== Deployment Pipeline Example
+
+You can connect the different tasks in a single workflow to get the full pipeline from the model training step to the model deployment and consumption steps. Each task will propagate the acquired variables to the connected tasks.
```

Comment:
`acquired` => `given`

Reply:
It is more acquired than given. As I mean the resulted variables. I can use resulted to be more clear.

Reply:
ok, let keep `acquired`